app/views/items/show.html.erb

```diff
@@ -1,3 +1,4 @@
 <p>Item: <%=@item.name%></p>
 <p>Description: <%=@item.description%></p>
-<p>Current price: $<%="#{@item_price}".insert(-3, '.')%></p>
\ No newline at end of file
+<p>Current price: $<%="#{@item_price}".insert(-3, '.')%></p>
```

Comment:
Very curious what this line does! Would like to meet up at some point today so you'd teach me if you're willing to!